rollup.config.js

```diff
@@ -25,6 +25,7 @@ export default {
 		plugins: [
 			replace({
 				'process.browser': true,
+				'preventAssignment': true,
```

Comment:
one last thing. it's a bit funny to me that `preventAssignment` is being added in between variables to replace. how about putting it first so that it's alphabetical? also, it doesn't need to be in single quotes. the others are because they contain `.`

Reply:
Mixing together options and the values to be replaced seems like a weird API. Looking at https://github.com/rollup/plugins/blob/6bee5980155df7b73cfbd9746556517c8d7f0ad7/packages/replace/src/index.js#L17 it looks like this is supported, but that they are not removing `preventAssignment` from the object when constructing the object of values to replace. But in any case, it would be better to use the separate `values` option (which is an object of replacements like we're currently passing), and have that be alongside any other options we might want to specify.

Even once it comes time to remove `preventAssignment` when we upgrade to a newer version of the plugin, we should keep `values` as the top-level object, to encourage users to use this better syntax, where any options they might want to specify can be kept separate from the strings you want to replace.

Reply:
@Conduitry what do you suggest

Reply:
Agreed. I wonder why this isn't documented, the mixed-case api is bizzare.

@makuzaverite the suggestion is use the plugin as follows:

```js
replace({
  preventAssignment: true,
  values: {
    'process.browser': true,
    ...
  }
})
```